### DISCUSSION TITLE
JavDB似乎增加了机器验证步骤，现在无法从JavDB抓去信息了

### DISCUSSION THREAD

**Discussion by moordc** [Yuukiy/JavSP#313] [问答 | Q&A]
如题，希望可以优化，毕竟以前的标签都是从JavDB抓取的，🙇‍♂️🙇‍♂️🙇‍♂️

**Comment by qq-751544744** [comment]
希望增加 javDB刮削进入网站后等待6秒再进行刮削，因为有个人机验证 是一个五秒盾

**Comment by sacchi2019** [comment]
刮削不了了，，作者还维护吗？

**Comment by musnow** [comment]
可以将 配置文件->ProxyFree->javdb 更改为最新的网址即可解决这个问题，当前最新域名为[javdb367.com](https://javdb367.com/)

**Comment by gsxgmpkt9020** [comment]
添加flaresolverr服务器 能够解决吗？

https://github.com/FlareSolverr/FlareSolverr

FlareSolverr is a proxy server to bypass Cloudflare and DDoS-GUARD protection.
